internal/browser/executor.go

```diff
@@ -161,7 +162,7 @@ func SetCookie(c *http.Cookie) chromedp.Action {
 // Cookies sets the current list of cookies into the provided destination.
 func Cookies(dst *[]*http.Cookie) chromedp.Action {
 	return chromedp.ActionFunc(func(ctx context.Context) error {
-		networkCookies, err := network.GetAllCookies().Do(ctx)
+		networkCookies, err := storage.GetCookies().Do(ctx)
```

Comment:
network.GetAllCookies was deprecated and has been removed - storage.GetCookies() is the recommended replacement.

Reply:
sneaking changes into the deps update...

lgtm